packages/snaps-controllers/src/interface/SnapInterfaceController.ts

```diff
@@ -199,7 +200,7 @@ export class SnapInterfaceController extends BaseController<
       // possibly infinite.
       draftState.interfaces[id] = {
         snapId,
-        content: element,
+        content: castDraft(element),
```

Comment:
Explain please!

Reply:
"`SnapElement` is not assignable to `WritableDraft<SnapElement>`" or something like that.